apps/nav-veileders-flate/src/mocks/MockHandler.ts

```diff
@@ -193,6 +195,28 @@ export class MockHandler {
     return EMDASH
   }
 
+  getForslag(): Forslag[] {
+    if (this.statusType === DeltakerStatusType.DELTAR) {
+      const fremtidigDato = new Date()
+      fremtidigDato.setDate(fremtidigDato.getDate() + 10)
+      const sluttdato = dayjs(fremtidigDato).format('YYYY-MM-DD')
+      const forslag = {
+        id: uuidv4(),
+        opprettet: dayjs().format('YYYY-MM-DD'),
+        begrunnelse:
+          'Vi har kommet i gang, men ser at det er hensiktsmessig ' +
+          'å fortsette tett oppfølging nå når han er i gang med å kontakte de riktige arbeidsgiverne. ' +
+          'nå er det totalt sett to hundre tegn. Ja, det er det..',
+        endring: {
+          type: ForslagEndringType.ForlengDeltakelse,
```

Comment:
Her klager intellij på at ForslagEndringType ikke er assignable til ForslagEndringType.ForlengDeltakelse, og jeg skjønner ikke hvorfor den gjør det eller hva jeg må gjøre for at den skal slutte å klage. Alt ser ut til å virke da 🤷‍♀️

Reply:
Med ditt format forsvant denne :D